app/packs/src/apps/mydb/elements/details/VersionsTableChanges.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import VersionsTableFields from 'src/apps/mydb/elements/details/VersionsTableFields';
+import VersionsTableModal from 'src/apps/mydb/elements/details/VersionsTableModal';
+import { Alert } from 'react-bootstrap';
+
+function VersionsTableChanges(props) {
+  const {
+    id, changes, handleRevert, isEdited
+  } = props;
+
+  const revertibleFields = () => {
+    if (isEdited) return -1;
+
+    let filteredFields = [];
+
+    changes.forEach(({ fields }) => {
+      filteredFields = filteredFields.concat(
+        fields.filter((field) => (field.revert.length > 0))
+      );
+    });
+
+    if (filteredFields.length === 0) {
+      return -2;
+    }
+
+    filteredFields = filteredFields.filter((field) => (field.currentValue !== field.oldValue));
+
+    return filteredFields.length;
+  };
+
+  const change = changes.map(({ name, fields }, index) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <React.Fragment key={index}>
+      <ol className="history-table-breadcrumb">
+        {name.map((item) => (
+          <li key={item} className="history-table-breadcrumb__element">
+            {item}
+          </li>
+        ))}
+      </ol>
+      <VersionsTableFields fields={fields} renderRevertView={false} />
+    </React.Fragment>
+  ));
+
+  const isRevertible = revertibleFields();
+  let alertText = 'You cannot undo changes. ';
+  if (isRevertible === 0) {
+    alertText += 'Every change is either up to date or irreversible';
```

Comment:
Now the first and last conditional branch both contain the "all changes are irreversible" case.